INSTALL.md

```diff
@@ -387,6 +388,24 @@ autoreconf -if
 make -j$(nproc)
 sudo make install
 cd ..
+
+
+Conversely, if you wish to explictly statically  or dynamically link sqlcipher use:
+
+Statically Linked:
+cd sqlpcipher
+$ ./configure --enable-tempstore=yes CFLAGS="-DSQLITE_HAS_CODEC" \
+    LDFLAGS="/opt/local/lib/libcrypto.a"
+$ make
+$ sudo make install
```

Comment:
Could you remove all the unnecessary `$` ?

Also, put things that aren't commands outside of code block – each variant should be in a separate code block.